beaker/cache.py

```diff
@@ -595,6 +595,8 @@ def cached(*args, **kwargs):
 
             def go():
                 return func(*args, **kwargs)
+            # save org function name
+            go.__name__ = func.__name__
```

Comment:
Yes, I see the value for debugging/logging. Just want to have a naming that doesn't collide with the original function name to avoid confusion. If you want to adapt the patch to a version where the name is prefix with `cached_` or anything like that I'd gladly merge it